<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scriptEditor</class>
 <widget class="QMainWindow" name="scriptEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="executeAll_btn">
            <property name="toolTip">
             <string>Execute all code in current tab
</string>
            </property>
            <property name="text">
             <string>Execute All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="executeSel_btn">
            <property name="toolTip">
             <string>Execute selected code in current tab
</string>
            </property>
            <property name="text">
             <string>Execute Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearHistory_btn">
            <property name="toolTip">
             <string>Clear output window</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="out_ly"/>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <layout class="QVBoxLayout" name="in_ly"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="saveSeccion_act"/>
    <addaction name="save_act"/>
    <addaction name="load_act"/>
    <addaction name="separator"/>
    <addaction name="exit_act"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="openManual_act"/>
    <addaction name="shortcuts_act"/>
    <addaction name="donate_act"/>
    <addaction name="separator"/>
    <addaction name="about_act"/>
   </widget>
   <widget class="QMenu" name="tools_menu">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="tabToSpaces_act"/>
   </widget>
   <widget class="QMenu" name="options_menu">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="theme_menu">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="editTheme_act"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="theme_menu"/>
    <addaction name="settingsFile_act"/>
   </widget>
   <widget class="QMenu" name="run_menu">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="execAll_act"/>
    <addaction name="execSel_act"/>
    <addaction name="clearHistory_act"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="options_menu"/>
   <addaction name="run_menu"/>
   <addaction name="tools_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <action name="clearHistory_act">
   <property name="text">
    <string>Clear Output</string>
   </property>
  </action>
  <action name="save_act">
   <property name="text">
    <string>Save Script</string>
   </property>
  </action>
  <action name="load_act">
   <property name="text">
    <string>Load Script</string>
   </property>
  </action>
  <action name="exit_act">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="openManual_act">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="quickHelp_act">
   <property name="text">
    <string>Quick Help</string>
   </property>
  </action>
  <action name="saveSeccion_act">
   <property name="text">
    <string>Save session</string>
   </property>
  </action>
  <action name="tabToSpaces_act">
   <property name="text">
    <string>Tab to spaces</string>
   </property>
  </action>
  <action name="spacesToTabs_act">
   <property name="text">
    <string>Spaces to tab</string>
   </property>
  </action>
  <action name="settingsFile_act">
   <property name="text">
    <string>Open Settings Folder</string>
   </property>
  </action>
  <action name="editTheme_act">
   <property name="text">
    <string>Edit ...</string>
   </property>
  </action>
  <action name="shortcuts_act">
   <property name="text">
    <string>Show Shortcuts</string>
   </property>
  </action>
  <action name="donate_act">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="about_act">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="execAll_act">
   <property name="text">
    <string>Execute All</string>
   </property>
  </action>
  <action name="execSel_act">
   <property name="text">
    <string>Execute Selected</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
